library/cpp/iterator/concatenate.h

```diff
@@ -32,8 +32,8 @@ namespace NPrivate {
                 friend struct TConcatenatorWithIndex<I...>;
 
                 // important, that it is a static function, compiler better optimizes such code
-                template <std::size_t index = 0, typename TMaybeConstIteratorState>
-                static TValue GetCurrentValue(std::size_t position, TMaybeConstIteratorState& iterators) {
+                template <std::size_t index = 0, typename OptConstIteratorState>
+                static TValue GetCurrentValue(std::size_t position, OptConstIteratorState& iterators) {
```

Comment:
```suggestion
                template <std::size_t index = 0, typename TOptConstIteratorState>
                static TValue GetCurrentValue(std::size_t position, TOptConstIteratorState& iterators) {
```
По нашему стайлгайду все типы начинаются с T*

Reply:
Исправил, буду иметь в виду.